geps/gep-1742.md

```diff
@@ -403,13 +401,22 @@ type HTTPRouteTimeouts struct {
 	// request stream has been received instead of immediately after the transaction is
 	// initiated by the client.
 	//
+	// This value follows a subset of Golang and XML Duration formatting, for example 1m, 1s,
+	// or 1ms. This value MUST BE >= 0 and <= 9999h. When "0" is specified, it indicates that
```

Comment:
Hour is not an accepted value in your kubebuilder validation pattern.  9999h is 599940m, so if you want to allow that amount, you would need to change the pattern to allow 6 digits.

Reply:
Good catch, thanks! This is an artifact of when I originally had hours as an allowed unit, will update pending the conversation above about whether or not we should add that back.